schedule/schedule.tpl

```diff
@@ -114,6 +114,7 @@
                     <a class="session-link" href="#{{session_id}}">
                       <i class="fa fa-link"></i>
                     </a>
+                    <input class="inputbox" type="text" onclick="this.select()" value="https://opentechevent.net/programm/#{{session_id}}" style="display:none;">
```

Comment:
Corrected that https://github.com/OpenTechSummit/open-event-scraper/pull/14/files